build.sh

```diff
@@ -134,6 +134,28 @@ function build_all_stretch_slim {
     echo "slim"
 }
 
+function build_kudulite {
+    local name=$1
+    local tag=$2
+    local base_dir=$DIR/kudulite
+
+    if [ -z "$name" ]; then
+        name=kudulite
+    fi
+
+    if [ -z "$tag" ]; then
+        tag=latest
+    fi
+
+    current_image=$ACR/$ACR_NAMESPACE/$name:$tag
+    echo -e "${CONSOLE_BOLD}${COLOR_GREEN}: Building $current_image ${CONSOLE_RESET}"
+    docker build -t $current_image -f $base_dir/Dockerfile $base_dir
+
+    if [[ "$tag" == 1\.*\.* ]]; then
```

Comment:
today that tag comes from the release on github. Do you plan to cut releases with 1.0.xxx for kudu separately?

Reply:
oh sorry I think you're passing that into your bash function. never mind.